plutarch-api/src/Plutarch/Api.hs

```diff
@@ -1273,3 +1371,10 @@ pmaybeLT whenBothNothing ltF = phoistAcyclic $
             $ pconstant whenBothNothing
         )
       $ pconstant True
+
+ptryFromData ::
+  forall (a :: S -> Type) (s :: S).
+  PTryFrom PData (PAsData a) =>
+  Term s PData ->
+  Term s (PAsData a)
+ptryFromData x = unTermCont $ fst <$> tcont (ptryFrom @(PAsData a) x)
```

Comment:
I don't think we need TermCont here

Reply:
It's a direct copy. I'll rewrite it.

Reply:
I don't think we need this helper really